modules/quant_api/src/EventSubscriber/QuantApi.php

```diff
@@ -177,14 +178,46 @@ public function onOutput(QuantEvent $event) {
         $fileOnDisk = str_replace('/system/files', $privatePath, $file);
       }
 
+      // Check if the file has changed.
       if (isset($item['existing_md5'])) {
         if (file_exists($fileOnDisk) && md5_file($fileOnDisk) == $item['existing_md5']) {
           continue;
         }
       }
 
-      // If the file exists we send it directly to quant otherwise we add it
-      // to the queue to generate assets on the next run.
+      // In Drupal 10.1, work around CSS/JS aggregation issues.
+      // Only process internal CSS/JS files.
+      $check_file = preg_match('/\.(css|js)$/', $file) && (strpos($item['original_path'], "http") !== 0);
```

Comment:
I'm sure the `preg_matc`h could be updated to also check the `http` but didn't want to fuss with it... feel free to suggest a better regex.